bundle/edu.gemini.shared.util/src/main/scala/edu/gemini/shared/util/DateTimeUtils.scala

```diff
@@ -0,0 +1,86 @@
+package edu.gemini.shared.util
+
+import java.time.{Instant, ZoneId, ZonedDateTime}
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+import java.util.{Comparator, Date, TimeZone}
+import java.util.function.ToLongFunction
+
+import scala.concurrent.duration._
+import scalaz._
+import Scalaz._
+
+object DateTimeUtils {
+  val UTC: ZoneId         = ZoneId.of("UTC")
+  val DefaultZone: ZoneId = ZoneId.systemDefault()
```

Comment:
This will use the timezone of the system this is being run. e.g. in the case of the OT it will be the user's TZ. This maybe what you intend but I just want to be sure as the result could be surprising

Reply:
Yes, this is actually deliberate, and has been extracted from a couple of places in the code.

Reply:
Can we rename this `SystemDefault` so it's a little bit more clear what it means? It should rarely be used.

Reply:
@tpolecat That makes more sense. I'll do that.